integreat_cms/core/management/commands/send_push_notifications.py

```diff
@@ -51,14 +51,27 @@ def handle(self, *args: Any, **options: Any) -> None:
                     f"The system runs with DEBUG=True but the region with TEST_REGION_SLUG={settings.TEST_REGION_SLUG} does not exist."
                 ) from e
 
-        pending_push_notifications = PushNotification.objects.filter(
+        retain_time = settings.FCM_NOTIFICATION_RETAIN_TIME_IN_HOURS
+
+        scheduled_push_notifications = PushNotification.objects.filter(
             scheduled_send_date__isnull=False,
             sent_date__isnull=True,
             draft=False,
             scheduled_send_date__lte=timezone.now(),
-            scheduled_send_date__gte=timezone.now()
-            - timedelta(hours=settings.FCM_NOTIFICATION_RETAIN_TIME_IN_HOURS),
+            scheduled_send_date__gte=timezone.now() - timedelta(hours=retain_time),
+        )
+
+        failed_push_notifications = PushNotification.objects.filter(
+            draft=False,
```

Comment:
```suggestion
            draft=False,
            is_template=False
```

Even though templates are always draft, I'd include this just to be more explicit